static/js/TextRange.jsx

```diff
@@ -637,7 +637,7 @@ class TextSegment extends Component {
         {segmentNumber}
         {linkCountElement}
         <p className="segmentText">
-          <ContentText overrideLanguage={overrideLanguage} html={{"he": he+ " ", "en": en+ " " }} bilingualOrder={["he", "en"]}/>
+          <VersionContent overrideLanguage={overrideLanguage} html={{"he": he+ " ", "en": en+ " " }} bilingualOrder={["he", "en"]} textImageLoadCallback={this.props.placeSegmentNumbers}/>
```

Comment:
Why is this prop called `textImageLoadCallback`? Does `imageLoadCallback` make sense instead?